airflow/providers/amazon/aws/operators/emr.py

```diff
@@ -597,7 +608,7 @@ class EmrCreateJobFlowOperator(BaseOperator):
     template_ext: Sequence[str] = (".json",)
     template_fields_renderers = {"job_flow_overrides": "json"}
     ui_color = "#f9c915"
-    operator_extra_links = (EmrClusterLink(),)
+    operator_extra_links = (EmrLogsLink(),)
```

Comment:
why we are removing `EmrClusterLink`?

Reply:
There is a very good reason for it.   I messed up.  Adding it back now, thanks for the catch.